environment_conda.yml

```diff
@@ -10,6 +10,9 @@ dependencies:
   - pip:
       - hummingbot
       - streamlit
+      - psycopg2-binary
```

Comment:
check if it's necessary to add since we already have sqlalchemy. Can be required as a dependency to make it work.